specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,575 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-12-01",
+    "title": "WebApplicationFirewallManagement",
+    "description": "APIs to manage web application firewall rules"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies": {
+      "get": {
+        "tags": [
+          "ListWebApplicationFirewallPolicies"
```

Comment:
`tags` should be names after resource type (e.g. `WebApplicationFirewallPolicies`). The same comment applies to all other `tags` in this file